cli-feed-v3.json

```diff
@@ -17,15 +17,15 @@
       "hidden": true
     },
     "v2": {
-      "release": "2.1.0",
+      "release": "2.2.2",

```

Comment:
We should schedule an OGF run before updating the prod tag, as there are multiple changes in both CLI and templates. Can you revert the change in this line? I can start the set up the OGF run.

Reply:
sure, I revert this one and only left the prerelease one